app/src/ui/repository.tsx

```diff
@@ -85,10 +86,7 @@ export class RepositoryView extends React.Component<
         <span className="with-indicator">
           <span>Changes</span>
           {hasChanges ? (
-            <Octicon
-              className="indicator"
-              symbol={OcticonSymbol.primitiveDot}
-            />
+            <FilesChangedBadge numFilesChanged={numFilesChanged} />
```

Comment:
@iAmWillShepherd @yongdamsh any objections to feature flagging this as it's slated for `1.3.0`? We might want to do a point release for the minification issues we uncovered with the Webpack migration.

Reply:
@iAmWillShepherd Should I use `enableNotificationOfBranchUpdates` feature flag added in #4769  to enable this change?

Reply:
I applied it in 9f7248f.